index.bs

```diff
@@ -3457,6 +3494,29 @@ To <dfn noexport>set an OS-support header</dfn> given a [=header list=]
 1. [=header list/Set a structured field value=] given
     ("<code>[=Attribution-Reporting-Support=]</code>", |dict|) in |headers|.
 
+To <dfn>obtain and deliver debug reports on OS registrations</dfn> given an
+[=os debug data type=] |dataType|, a [=list=] of [=OS registrations=] |registrations|,
+and an [=origin=] |contextOrigin|:
+
+1. If |registrations| [=list/is empty=], return.
+1. Let |contextSite| be the result of [=obtaining a site=] from |contextOrigin|.
+1. [=list/iterate|For each=] |registration| of |registrations|:
+    1. If |registration|'s [=OS registration/debug reporting enabled=] is false, return.
+    1. Let |origin| be |registration|'s [=OS registration/url=]'s [=url/origin=].
+    1. If |origin| is not [=check if an origin is suitable|suitable=], return.
```

Comment:
Shouldn't these returns be `[=iteration/continue=]`?

Reply:
Yes, a leftover from a refactor. Thanks.